newlib/libc/tinystdio/stdio.h

```diff
@@ -300,14 +300,14 @@ int	ferror(FILE *__stream);
 #endif
 
 __extension__ typedef _fpos_t fpos_t;
-int fgetpos(FILE *stream, fpos_t *pos);
+int fgetpos(FILE *__restrict fp, _fpos_t *__restrict pos);
```

Comment:
Do these need to use `_fpos_t` instead of `fpos_t`? Is `fpos_t` not supposed to be defined with `#include <stdio.h>`?

Reply:
Changed it to  
```c
int fgetpos(FILE * restrict stream, fpos_t * restrict pos);
```